README.md

```diff
@@ -1 +1,104 @@
-# angular-ui-router-readme
\ No newline at end of file
+# What is ui-router?
+
+## Overview
+
+`ngRoute` provides us with basic functionality, but the leading router for Angular is `uiRouter`. This provides much more, such as nested views, multiple views and a few different offerings. Let's take a look at how `uiRouter` does things instead.
+
+## Objectives
+
+- Describe ui-router and ui-view
+- Write a route using ui-router
+- Integrate a Controller with ui-router
+- Integrate a template with ui-router
+
```

Comment:
Add a description of nested views here. What are nested views, and why are they important? Then, we'll be excited to see how uiRouter can make our lives better.